### PR TITLE
Update SEO content, see MISC-119

### PR DESCRIPTION
**Pull request by ccedric**
Motivation:
- we need to make sure the title and description are correct for SEO purposes

Modifications:
- fixed typos and add missing leads

### CODE REVIEW

**Comment by tpetillot** [comment]
Could you also add description on nodes too (`_index.md`)

**Comment by ccedric** [comment]
> Could you also add description on nodes too (`_index.md`)
ah, didn't know we needed that

File: content/docs/plugins/bamboo/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Bamboo Plugin"
-description: "Learn how to configure Bamboo plugin and integrate your simulations."
-lead: "Integrate your simulations to your Bamboo CI."
+description: "Learn how to configure the FrontLine Bamboo plugin and run your simulations."
+lead: "Run your FrontLine simulations from your Bamboo CI."
```
Comment (tpetillot) [inline content/docs/plugins/bamboo/index.md:4]: Wondering about this suggestion:
```suggestion
lead: Run your FrontLine simulations from your Bamboo CI.
```
For consistency with cases like:
```yaml
    pre: >-
      <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0
      0 24 24" fill="none" stroke="currentColor" stroke-width="2"
      stroke-linecap="round" stroke-linejoin="round" class="feather
      feather-twitter"><path d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0
      0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0
      20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"></path></svg>
```

**Comment by ccedric** [comment]
closing, as it is on the wrong branch

**Pull request closed without merging**